tests/single-node/test.sh

```diff
@@ -19,8 +19,10 @@ setup_suite() {
         sudo losetup /dev/loop0 /kube-lvm || die
 
         echo "Disabling iptables"
-        sudo systemctl disable --now iptables || die
-        sudo systemctl disable --now ip6tables || die
+        if systemctl is-enabled --quiet iptables; then
+          sudo systemctl disable --now iptables || die
```

Comment:
I have a question for you @Zempashi, I checked out a VM on rackspace and the services were not enabled, should we still keep those two lines?

Reply:
ok to me